specification/application_layer/conventions.tex

```diff
@@ -3,7 +3,7 @@ \section{Application-level conventions}\label{sec:application_level_conventions}
 This section describes a set of high-level conventions designed to enhance compatibility
 of applications leveraging UAVCAN.
 The conventions described here are not mandatory to follow;
-however, every deviation should be justified and adequately documented.
+however, every deviation should be justified and adequately documented. % This may be outside the scope of this document, but what constitutes "adequately documented"? This may well be explained elsewhere.
```

Comment:
```suggestion
however, every deviation should be justified and documented.
```

Reply:
What we're doing here in this sentence is just advising the users on what the best practice should be. We want to ensure that their engineering approaches are solid and that if their implementations are re-used by other UAVCAN adopters, they do not stumble upon obscure compatibility issues. This is not really about the spec doc at hand; rather, it's about the larger ecosystem out there. Instead of trying to spec out everything we might want to release it in the current minimal form and then see if the ecosystem needs extra control input as we go.

Reply:
Sounds like a good plan!